app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,29 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="#">Brand</a>
+    </div>
+
+    <div class="collapse navbar-collapse">
+      <ul class="nav navbar-nav navbar-right">
+        <% if user_signed_in? %>
+          <li><a href="#" class="navbar-link no-pointer">Welcome, <%= current_user.name %></a></li>
```

Comment:
For the UX, would you like the user's name to be presented in a certain _case_, regardless of how the user enters in their name?  

For example, if the user enters their name as `john brown`, how do you want that to be presented in the nav bar, with regards to case?  Perhaps [titleize](https://apidock.com/rails/String/titleize)?